src/docfx/build/page/AttributeTransformer.cs

```diff
@@ -45,7 +45,8 @@ private static object TransformContent(Context context, DataTypeAttribute attrib
         {
             if (attribute is HrefAttribute)
             {
-                var (error, link, _) = context.DependencyResolver.ResolveLink((string)value, file, file, buildChild);
+                var (error, link, _) = context.DependencyResolver.ResolveLink((string)value, file, file, buildChild, file.FilePath.EndsWith("index.yml"));
```

Comment:
file.Schema.Name == "LandingData"

Reply:
right

Reply:
Actually, we could not use `file.Schema.Name == "LandingData"` here, because `file.Schame` could be null and we are getting `documentType` instead as the test shows.